components/fetch-build-scan-data-cmdline-tool/build.gradle.kts

```diff
@@ -1,5 +1,8 @@
+@file:Suppress("UnstableApiUsage")
+
 plugins {
     java
+    `jvm-test-suite`
```

Comment:
I would use the ticks for java and application, too, just to keep the visuals more similar right here.

Reply:
IntelliJ will complain about unnecessary backticks if you do that.

Unfortunately, this is idiomatic Kotlin DSL. 

That being said, I also dislike it and am not a fan of the inconsistency. In fact, for this very reason, I do it the same way as would be done in Groovy in my personal repositories. For example:

```kotlin
plugins {
    id("application")
    id("java")
    id("jvm-test-suite")
    id("com.github.johnrengelman.shadow") version "8.1.1"
    id("org.openapi.generator") version "6.5.0"
}
```

It's also not idiomatic Kotlin DSL, but at least it is consistent and IntelliJ does not complain.

Reply:
Good compromise